system/Database/BaseBuilder.php

```diff
@@ -3040,7 +3040,7 @@ protected function compileSelect($selectOverride = false): string
 				. $this->compileWhereHaving('QBHaving')
 				. $this->compileOrderBy(); // ORDER BY
 		// LIMIT
-		if ($this->QBLimit)
+		if ($this->QBLimit !== false)
```

Comment:
```suggestion
		if (false !== $this->QBLimit)
```

Reply:
I think we do this all over the framework in that order now.

Reply:
Are you sure? I searched the codebase and it's mostly done with false at the end (140+ vs. 9). Same for "=== false" (150+ vs 17). But I can switch it around of course! :slightly_smiling_face: Will do so when the tests pass as I just added more of this :sweat_smile:

Reply:
Oh okay 😆 
Maybe just wait with that then. If it is desired, someone from the team will mention it after everything works.

Reply:
We don't do yoda conditions. :)

Reply:
Okay my bad. Thought I saw it in a recent PR.